src/frontend-admin/src/components/App/App.tsx

```diff
@@ -20,6 +22,62 @@ export const App: React.FC = () => {
           Learn React
         </a>
       </header>
-    </div>
+  </div>*/
+    <div>
+      {/*header*/}
+
+      <Router>
+      
+        <div className={styles.navContainer}>
+          <div className={styles.space} />
```

Comment:
I always recommend using really descriptive class names - I like `navContainer` since I immediately know looking at the name what part of the component it represents. `space` however sticks out to me as kinda vague. If this class is intended to add some styling to the navbar, then perhaps it belongs as a class (with a more descriptive name) on the `<nav>` tag. Make sense?

Reply:
Yeah, that makes sense. The space div was just the empty space above the navbar but it probably makes more sense just to position the navbar lower and remove 'space'.